CHANGELOG.md

```diff
@@ -0,0 +1,40 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [7.1.0] 2022-07-01
+
+* Try to establish connection against IPFS node several times before shutting down the service.
```

Comment:
Why removing pull request links?

Reply:
I went for the quick and readable format.
PR links can be added again.
They will look like [PR-NUMBER](PR-LINK) which will modify the file readability.
We will need preview tools in this case, which goes with your suggestion on table formatting.